projects/js-packages/publicize-components/src/components/settings-button/index.js

```diff
@@ -4,8 +4,13 @@
  * Component which allows user to click to open settings
  * in a new window/tab.
  */
+import { ThemeProvider } from '@automattic/jetpack-components';
+import { useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
+import { useReducer, useState } from 'react';
 import usePublicizeConfig from '../../hooks/use-publicize-config';
+import { SOCIAL_STORE_ID } from '../../social-store';
```

Comment:
Can we please use `store` instead of store ID?

That helps in auto completions and can, at times, catch typos easily.